common_features.mk

```diff
@@ -604,7 +607,6 @@ ifeq ($(strip $(UNICODE_ENABLE)), yes)
 endif
 
 ifeq ($(strip $(UNICODE_COMMON)), yes)
-    OPT_DEFS += -DUNICODE_COMMON_ENABLE
```

Comment:
```suggestion
    OPT_DEFS += -DUNICODE_COMMON_ENABLE
```

Reply:
huh wasn't able to commit this in the default way so I have added that line back in
(this was some change on the development branch that I accidentally carried foreword my bad)